decidim-core/lib/decidim/participable.rb

```diff
@@ -64,6 +64,10 @@ def has_steps?
       def manifest
         self.class.participatory_space_manifest
       end
+
+      def self.non_private_spaces
+        where(private_space: false)
```

Comment:
Models implementing the `Decidim::Participable` concern are not required to include the `Decidim::HasPrivateUsers` one, so they might not have the `private_space` column and this will raise an error.

Reply:
I've moved inside `HasPrivateUsers`